### PR TITLE
Localization for the Help Command

### PR DESCRIPTION
**Pull request by vsccarl**
Added localization for strings in the help command.

### CODE REVIEW

File: src/dotnet/commands/dotnet-help/HelpCommand.cs [hunk 1]
```diff
@@ -8,40 +8,40 @@ namespace Microsoft.DotNet.Tools.Help
...
+  build         {LocalizableStrings.BuildDefinition}
+  publish       {LocalizableStrings.PublishDefinition}
+  run           {LocalizableStrings.RunDefinition}
+  test          {LocalizableStrings.TestDefinition}
+  pack          {LocalizableStrings.PackDefinition}
+  migrate       {LocalizableStrings.MigrateDefinition}
 
-Project modification commands:
-  add           Add items to the project
+{LocalizableStrings.ProjModificationCommands}:
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-help/HelpCommand.cs]: Just rename this to ProjectModificationCommands. The proj abbreviation is not really doing much for us.

**Comment by piotrpMSFT** [comment]
@MattGertz can you pre-approve this change so we can merge as soon as it is green?

**Comment by piotrpMSFT** [comment]
@vsccarl this PR is failing with `
error CS0133: The expression being assigned to 'HelpCommand.UsageText' must be constant `. Can you address?

**Comment by MattGertz** [comment]
Approved pending tests passing.

**Comment by piotrpMSFT** [comment]
I'm fixing up this PR now.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX x64 Release Build please.

**Pull request merged**